portal-frontend/src/app/features/review-submission/review-submission.component.ts

```diff
@@ -97,7 +97,7 @@ export class ReviewSubmissionComponent implements OnInit, OnDestroy {
               // setTimeout is required for stepper to be initialized
               setTimeout(() => {
                 const stepInt = parseInt(stepInd);
-                this.customStepper.navigateToStep(stepInt, true);
```

Comment:
This will ignore if the step is provided in URL. Is it for testing purposes?

Reply:
Interestingly enough this only seems to break navigating to a specific parcel. Everything else seems fine.

Reply:
I found it odd that the review page had its own call to change steps but looked like it was the only place.